02-household-queries/chainlit-household-bot.py

```diff
@@ -25,6 +39,7 @@ async def init_chat():
                 value="chooseBetter",
                 label="Demo choosing better response",
             ),
+            cl.Action(name="uploadQueryAct", value="upload_query", label="Upload query")
         ],
```

Comment:
Add an action to load the default `tanf.pdf` and `guru_cards_for_nava.json`. I don't want to have to upload it every time.